DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1003 @@ Live discussion in #document-authoring on Slack.
+	* `@value` set to the `Period@id` of the first period.
+
+Note: Not all [=representations=] in an [=adaptation set=] need to be period-connected. For example, if a new [=period=] is introduced to add a [=representation=] that contains a new video quality level, all other [=representations=] will likely be connected but not the one that was added.
+
+<figure>
+	<img src="Images/Timing/SegmentOverlapOnPeriodConnectivity.png" />
+	<figcaption>The same [=media segment=] will often exist in two periods at a period-connected transition. On the diagram, this is segment 4.</figcaption>
+</figure>
+
+As a [=period=] may start and/or end in the middle of a [=media segment=], the same [=media segment=] MAY simultaneously exist in two period-connected [=representations=], with one part of it scheduled for playback during the first [=period=] and the other part during the second [=period=]. This is likely to be the case when no [=sample timeline=] discontinuity is introduced by the transition.
+
+Clients SHOULD NOT present a [=media segment=] twice when it occurs on both sides of a period transition in a period-connected [=representation=].
+
+Clients SHOULD ensure seamless playback of period-connected [=representations=] in consecutive [=periods=].
+
+Note: The exact mechanism that ensures seamless playback depends on client capabilities and will be implementation-specific. Any shared [=media segment=] overlapping the [=period=] boundary may need to be detected and deduplicated to avoid presenting it twice.
+
+### Period continuity ### {#timing-connectivity-continuity}
+
+In addition to [[#timing-connectivity|period connectivity]], [[!MPEGDASH]] defines period continuity, which is a special case of period connectivity where the two samples on the boundary between the connected [=representations=] are consecutive on the same [=sample timeline=].
+
+Note: The above can only be true if the sample boundary exactly matches the period boundary.
+
+Period continuity MAY be signaled in the MPD when the above condition is met, in which case period connectivity SHALL NOT be simultaneously signaled on the same [=representation=]. Continuity implies connectivity.
+
+The signaling of period continuity is the same as for [[#timing-connectivity|period connectivity]], except that the value to use for `@schemeIdUri` is `urn:mpeg:dash:period-continuity:2015`.
+
+Clients MAY take advantage of any platform-specific optimizations for seamless playback that knowledge of period continuity enables; beyond that, clients SHALL treat continuity the same as connectivity.
+
+## Dynamic MPDs ## {#timing-dynamic}
+
+This section only applies to dynamic MPDs.
+
+Advisement: A dynamic MPD must conform to the constraints in this document not only at its moment of initial publishing but through the entire validity duration of the MPD (as defined by `MPD@minimumUpdatePeriod`).
+
+## Real time clock synchronization ## {#timing-sync}
```

Comment:
Is it intended to have `Real time clock synchronization` and `Time shift window` as independent sections? As they only apply to dynamic MPDs should they be subsections?

Reply:
Yes, error.